app/controllers/sections/consignee/ConsigneeIndexController.scala

```diff
@@ -75,16 +75,28 @@ class ConsigneeIndexController @Inject()(override val messagesApi: MessagesApi,
     val gbwkAndSpecificDestinationTypes: Boolean =
       userTypeFromErn == GreatBritainWarehouseKeeper && destinationTypePageAnswer == ExportWithCustomsDeclarationLodgedInTheUk
 
+    val gbrcAndSpecificDestinationTypes: Boolean =
+      userTypeFromErn == GreatBritainRegisteredConsignor && destinationTypePageAnswer == ExportWithCustomsDeclarationLodgedInTheUk
+
     val xiwkAndSpecificDestinationTypes: Boolean = {
       val validDestinationTypes: Seq[MovementScenario] = Seq(
-        RegisteredConsignee,
         ExportWithCustomsDeclarationLodgedInTheUk,
         ExportWithCustomsDeclarationLodgedInTheEu
       )
+
       userTypeFromErn == NorthernIrelandWarehouseKeeper && validDestinationTypes.contains(destinationTypePageAnswer)
     }
 
-    gbwkAndSpecificDestinationTypes || xiwkAndSpecificDestinationTypes
+    val xircAndSpecificDestinationTypes: Boolean = {
+      val validDestinationTypes: Seq[MovementScenario] = Seq(
+        ExportWithCustomsDeclarationLodgedInTheUk,
+        ExportWithCustomsDeclarationLodgedInTheEu
+      )
+
+      userTypeFromErn == NorthernIrelandRegisteredConsignor && validDestinationTypes.contains(destinationTypePageAnswer)
+    }
+
+    gbwkAndSpecificDestinationTypes || gbrcAndSpecificDestinationTypes || xiwkAndSpecificDestinationTypes || xircAndSpecificDestinationTypes
```

Comment:
The logic in the copydeck has been made more abstract. All the matching is done on the destination type only so the user type doesn't need to be considered

Reply:
May be worth double checking its safe to remove any user type checking with UCD